gprofiler/merge.py

```diff
@@ -31,6 +32,20 @@
 FRAME_REGEX = re.compile(r"^\s*[0-9a-f]+ (.*?) \((.*)\)$")
 
 
+class EnrichmentOptions:
```

Comment:
dataclass

Reply:
It's a single instance. What it does matter for? shortening the code?

Reply:
You can many benefits ootb but yeah, shortening code

Reply:
Made it a dataclass for easy printing